docs/reference/argocd.md

```diff
@@ -733,6 +733,8 @@ Name | Default | Description
 Resources | [Empty] | The container compute resources.
 MountSAToken | false | Whether the ServiceAccount token should be mounted to the repo-server pod.
 ServiceAccount | "" | The name of the ServiceAccount to use with the repo-server pod.
+StrictTLS | false | Whether to enforce strict TLS checking on all components when communicating with repo server
```

Comment:
This is really `TLSVerify`, right?  Enable to validation TLS cerificate.

Reply:
Yes, I do agree with you, @wtam2018. Will change it to `TLSVerify`.

Reply:
Changed it to `VerifyTLS` instead.